app/models/asylum_support_decision.rb

```diff
@@ -1,19 +1,19 @@
 class AsylumSupportDecision < Document
-  validates :tribunal_decision_category, presence: true
+  validates :tribunal_decision_categories, presence: true
   validates :tribunal_decision_decision_date, presence: true, date: true
   validates :tribunal_decision_judges, presence: true
   validates :tribunal_decision_landmark, presence: true
   validates :tribunal_decision_reference_number, presence: true
-  validates :tribunal_decision_sub_category, presence: true, asylum_support_decision_sub_category: true
+  validates :tribunal_decision_sub_categories, presence: true, asylum_support_decision_sub_category: true
 
   FORMAT_SPECIFIC_FIELDS = %i(
       hidden_indexable_content
-      tribunal_decision_category
+      tribunal_decision_categories
```

Comment:
If you keep this `tribunal_decision_category` and `tribunal_decision_sub_category` here temporarily, we can use it for the migration.